draft-liu-teas-transport-network-slice-yang.md

```diff
@@ -0,0 +1,386 @@
+   mechanism is to guarantee the information isolation among slice
+   tenants and to avoid resource conflicts. The access to slice
+   management functions will only be permitted after successful security
+   checks.
+
+# Security Considerations
+
+   The YANG module specified in this document defines a schema for data
+   that is designed to be accessed via network management protocols such
+   as NETCONF {{!RFC6241}} or RESTCONF {{!RFC8040}}.  The lowest NETCONF layer
+   is the secure transport layer, and the mandatory-to-implement secure
+   transport is Secure Shell (SSH) {{!RFC6242}}.  The lowest RESTCONF layer
+   is HTTPS, and the mandatory-to-implement secure transport is TLS
+   {{!RFC8446}}.
+
+   The NETCONF access control model {{!RFC8341}} provides the means to
+   restrict access for particular NETCONF or RESTCONF users to a
+   preconfigured subset of all available NETCONF or RESTCONF protocol
+   operations and content.
+
+   There are a number of data nodes defined in this YANG module that are
+   writable/creatable/deletable (i.e., config true, which is the
+   default).  These data nodes may be considered sensitive or vulnerable
+   in some network environments.  Write operations (e.g., edit-config)
+   to these data nodes without proper protection can have a negative
+   effect on network operations.  Considerations in Section 8 of
+   {{!RFC8795}} are also applicable to their subtrees in the module defined
+   in this document.
+
+   Some of the readable data nodes in this YANG module may be considered
+   sensitive or vulnerable in some network environments.  It is thus
+   important to control read access (e.g., via get, get-config, or
+   notification) to these data nodes.  Considerations in Section 8 of
+   {{!RFC8795}} are also applicable to their subtrees in the module defined
+   in this document.
+
+# IANA Considerations
+
+   It is proposed to IANA to assign new URIs from the "IETF XML
+   Registry" {{!RFC3688}} as follows:
+
+~~~~
+   URI: urn:ietf:params:xml:ns:yang:ietf-ns-topo
+   Registrant Contact: The IESG
+   XML: N/A; the requested URI is an XML namespace.
+~~~~
+
+   This document registers a YANG module in the YANG Module Names
+   registry {{!RFC6020}}.
+
+~~~~
+   name: ietf-ns-topo
+   namespace: urn:ietf:params:xml:ns:yang:ietf-ns-topo
+   prefix: ns-topo
+   reference: RFC XXXX
+~~~~
+
+--- back
+
+# Acknowledgments
+
+   The TEAS Network Slicing Design Team (NSDT) members included Aijun
+   Wang, Dong Jie, Eric Gray, Jari Arkko, Jeff Tantsura, John E Drake,
+   Luis M.  Contreras, Rakesh Gandhi, Ran Chen, Reza Rokui, Ricard
+   Vilalta, Ron Bonica, Sergio Belotti, Tomonobu Niwa, Xuesong Geng, and
+   Xufeng Liu.
+
+# Data Tree for the Example in Section 3.1
+
+## Native Topology
+   This section contains an example of an instance data tree in the JSON
+   encoding {{!RFC7951}}.  The example instantiates "ietf-network" for the
+   native topology depicted in {{fig-ns-topo-example}}.
```

Comment:
The existing ietf-ns-topo-example.json correctly shows an example of the native topoloy. There is no change needed. However, a native topology does not have attributes related to network slicing.  Section `# Model Applicability` also describes an example of customized topology "blue". The "blue" topology contains the attributes augmented for the network slicing. So I added a file `ietf-ns-topo-blue.json` to the repo to show the example.